app/screens/send-bitcoin-screen/send-bitcoin-completed-screen.tsx

```diff
@@ -114,6 +116,52 @@ const SendBitcoinCompletedScreen: React.FC<Props> = ({ route }) => {
     }
   }, [client, feedbackModalShown, LL, showSuggestionModal, navigation, requestFeedback])
 
+  const copyToClipboard = (text: string, message: string) => {
+    Clipboard.setString(text)
+    toastShow({
+      type: "success",
+      message,
+      LL,
+    })
+  }
+
+  const SuccessActionComponent = () => {
+    if (!successAction) return null
+
+    const originalText = successAction.message ?? ""
+    const truncatedText =
+      originalText.slice(0, 300) + (originalText.length > 300 ? "..." : "")
+
+    switch (successAction.tag) {
+      case "message":
```

Comment:
it must have 3 cases where are url and aes? please add them with proper testing. it can be an util or a component (please check __tests__ so you can get an idea on how to implement it)